src/API/Logs/LoggerInterface.php

```diff
@@ -12,6 +14,20 @@ interface LoggerInterface
      */
     public function emit(LogRecord $logRecord): void;
 
+    /**
+     * @see https://github.com/open-telemetry/opentelemetry-specification/blob/v1.40.0/specification/logs/api.md#emit-an-event
+     */
+    public function emitEvent(
```

Comment:
We shouldn't add this method until it is defined by the spec.
(Was removed in https://github.com/open-telemetry/opentelemetry-specification/pull/4352)

Reply:
Good catch. I'll remove this portion for now.